mood-analysis.rb

```diff
@@ -19,6 +20,58 @@ def analyze_mood(words)
   return ":-|"
 end
 
+def strip_punctuation(words)
+  words.gsub(/[!.#,]/, "")
+end
+
+def happy_days(array)
+  entries = 0
+  happy_faces = 0
+  array.each do |line|
+    if happy_faces < 3
+      emotion = analyze_mood(line)
+      entries += 1
+      if emotion == ":-)"
+      happy_faces += 1
+      break if happy_faces == 3
```

Comment:
There's no reason to do a break here and then later check for this condition again later to do the return. Instead, just return here.